.grabl/automation.yml

```diff
@@ -13,6 +17,10 @@ build:
         SONARCLOUD_CODE_ANALYSIS_CREDENTIAL=$SONARCLOUD_CREDENTIAL \
            bazel run @graknlabs_dependencies//tool/sonarcloud:code-analysis -- \
            --project-key graknlabs_console --branch=$GRABL_BRANCH --commit-id=$GRABL_COMMIT
+    dependency-analysis:
+      machine: graknlabs-ubuntu-20.04
```

Comment:
why the machine is not graknlabs-ubuntu-20.04-java11 ?

Reply:
@VladGan I've used the same image as used in another jobs; later we might either upgrade it to `-java11` or (better) make the base image contain Java 11